src/api/apiCore.cpp

```diff
@@ -153,12 +155,13 @@ RGL_API rgl_status_t rgl_cleanup(void)
 	return status;
 }
 
-void TapePlayer::tape_cleanup(const YAML::Node& yamlNode)
+void TapeCore::tape_cleanup(const YAML::Node& yamlNode, TapeState& tapeState)
 {
 	rgl_cleanup();
-	tapeMeshes.clear();
-	tapeEntities.clear();
-	tapeNodes.clear();
+	tapeState.meshes.clear();
+	tapeState.entities.clear();
+	tapeState.nodes.clear();
+	tapeState.textures.clear();
```

Comment:
:1st_place_medal:

Reply:
I have created a method `clear` in `PlaybackState` to remember about clearing when a new TapeAPIObject will be added. When `textures` were added, we forgot to clear them in `tape_cleanup` (`rgl_cleanup` playback function).